omnigibson/object_states/attached_to.py

```diff
@@ -76,10 +76,10 @@ def remove(self):
         og.sim.remove_callback_on_stop(name=f"{self.obj.name}_detach")
 
     @classproperty
-    def metalink_prefix(cls):
+    def meta_link_type(cls):
         """
         Returns:
-            str: Unique keyword that defines the metalink associated with this object state
+            str: Unique keyword that defines the meta link associated with this object state
         """
         return m.ATTACHMENT_LINK_PREFIX
```

Comment:
should these macros still be called `XXX_PREFIX`?

Reply:
Changed.